compile/error.go

```diff
@@ -64,6 +64,14 @@ func (e includeAsDisabledError) Error() string {
 	return "include-as syntax is currently disabled"
 }
 
+// includingHyphenatedFileNotAllowed is raised when the user attempts to include hyphenated thrift file which is not
+// allowed
+type includingHyphenatedFileNotAllowed struct{}
+
+func (e includingHyphenatedFileNotAllowed) Error() string {
+	return "including hyphenated thrift file is not allowed"
```

Comment:
Let's be more consistent with the existing errors
```suggestion
	return "cannot include hyphenated Thrift files"
```